t/test_postgresql

```diff
@@ -1,11 +1,7 @@
 #!/bin/sh -e
 
-DIR=$1
-if test -z "$DIR"; then
-    DIR=$(mktemp -d)
-else
-    DIR=$(readlink -f "$DIR")
-fi
+DIR=${1:-/dev/shm/tpg}
```

Comment:
I don't recommend storing bigger data here. If really the only way, I recommend checking the available size of this path first before using it. A system with high utilization and low memory could yield out-of-space issues quite easily.

Reply:
This is only for tests and our usual CI systems already use the same path everywhere as that is the default used in the Makefile which is used by both developers as well as CI. This only makes a difference for "expert developers" calling the script manually so I don't think this is a relevant problem.